script.js

```diff
@@ -11,3 +11,79 @@ document.querySelectorAll('.nav-items').forEach((e) => e.addEventListener('click
   hambuger.classList.remove('active');
   hambuger.classList.remove('active');
 }));
+
+// Featured speakers section
+
+const speakers = [
+  {
+    id: 1,
+    featuredImage: 'lorians.JPG',
+    Name: 'Lorians Mercy Ogallo',
+    title: 'The CEO at Mochez model International',
+    description: 'Lorians is the founder of Mochez models and in charge of implementing the companys goals, mission and vision',
+    content: 'show',
+  },
+  {
+    id: 2,
+    featuredImage: 'nicole.jpg',
+    Name: 'Nicole Terry',
+    title: 'Health, nutritionist and lifestyle coach',
+    description: 'As a lifestyle coach, Terry educates individuals on the need to identify their dietary requirements and restrictions',
+    content: 'show',
+  },
+  {
+    id: 3,
+    featuredImage: 'pamela.jpg',
+    Name: 'Pamela Grace',
+    title: 'Psychologist leader',
+    description: 'Pamela is an experienced psychologist and a has published several books on dealing with anxiety and depression',
+    content: 'show',
+  },
+  {
+    id: 4,
+    featuredImage: 'patricia.jpg',
+    Name: 'Vanita Joy',
+    title: 'Photographer',
+    description: 'Vanita has been leading various photography projects and exhibition. Get a chance to learn from the best',
+    content: 'hide',
+    profileName: 'showSpeaker',
+  },
+  {
+    id: 5,
+    featuredImage: 'lorians.JPG',
+    Name: 'M. Quincy',
+    title: 'Designer',
+    description: '“Good design is actually a lot harder to notice than poor design, in part because good designs fit our needs so well that the design is invisible,”',
+    content: 'hide',
+    profileName: 'showSpeaker',
+  },
+  {
+    id: 6,
+    featuredImage: 'nicole.jpg',
+    Name: 'Patricia G',
+    title: 'Fitness Instructor',
+    description: '“To keep the body in good health is a duty… otherwise we shall not be able to keep our mind strong and clear.” “When it comes to eating right and exercising, there is no I will start tomorrow.” “It never gets easier, you just get better.”',
+    content: 'hide',
+    profileName: 'showSpeaker',
+  },
+];
+
+const allSpeakers = speakers.map((speaker) => `<div class="speaker ${speaker.content}${speaker.profileName}">
+    <img src="images/${speaker.featuredImage}" alt="">
```

Comment:
- [ ] OPTIONAL
It's a good practice to provide `alt` names for your images so as to improve the accessibility features of your site.  Kindly endeavor to add `alt` names to **all** your images.